backend/fms_core/template_importer/row_handlers/sample_submission/sample.py

```diff
@@ -59,10 +62,18 @@ def process_row_inner(self, sample, library, container, project, parent_containe
                                individual["reference_genome"],
                                mother_obj,
                                father_obj])
+        can_use_generic_individual = (any([taxon_obj is not None and taxon_obj.ncbi_id == self.HUMAN_TAXON_ID,
+                                           taxon_obj is not None and taxon_obj.ncbi_id == self.MOUSE_TAXON_ID,
+                                           reference_genome_obj is not None and reference_genome_obj.taxon.ncbi_id == self.HUMAN_TAXON_ID,
+                                           reference_genome_obj is not None and reference_genome_obj.taxon.ncbi_id == self.MOUSE_TAXON_ID])
```

Comment:
This one is especially problematic

Reply:
Can we somehow have can_use_generic_individual as a field/column for taxon or reference genome?

Reply:
That one is not problematic ... this is related to user input ... it is tied to the harcoded value but the harcoded value must not be can_use_generic_individual ... the fact there is a taxon_obj is due to user setting a value in a template column ...